<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph3D_AddNew_gui</class>
 <widget class="QDialog" name="Graph3D_AddNew_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Graph 3D</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mathgraphica.qrc">
    <normaloff>:/images/images/graf3d.png</normaloff>:/images/images/graf3d.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_fxy">
     <property name="text">
      <string>f(x,y) =</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit_fxy">
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_variables">
     <property name="text">
      <string>variables:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_x"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_y"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_x_plus">
     <property name="text">
      <string>+x =</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_xmax"/>
   </item>
   <item row="2" column="2" rowspan="5">
    <widget class="QLabel" name="label_image">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../mathgraphica.qrc">:/images/images/3D.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_x_minus">
     <property name="text">
      <string>-x =</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_xmin"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_y_plus">
     <property name="text">
      <string>+y =</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_ymax"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_y_minus">
     <property name="text">
      <string>-y =</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_ymin"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_delta">
     <property name="text">
      <string>dx,dy =</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEdit_dxdy"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Graph colors:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_colorA">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_colorB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_colorC">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_colorD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_Ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>pushButton_cancel</zorder>
     <zorder>pushButton_Ok</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mathgraphica.qrc"/>
 </resources>
 <connections/>
</ui>
